lib/android-controller.ts

```diff
@@ -583,6 +583,23 @@ export class AndroidController {
     private static gettokenPrefix(type: DeviceType) {
         return type === DeviceType.DEVICE ? "" : "emulator-";
     }
+
+    private static getAlwaysFinishActivitiesGlobalSettingsValue(device: IDevice): boolean {
+        const prefix = AndroidController.gettokenPrefix(device.type);
+        return executeCommand(AndroidController.ADB + " -s " + `${prefix}${device.token}` + " shell settings get global always_finish_activities").trim() === "1";
```

Comment:
You can move the command to a variable.
Also you can move the result of the execution to a variable.
```
AndroidController.ADB + " -s " + `${prefix}${device.token}` + " shell settings get global always_finish_activities"
```
-->
```
`${AndroidController.ADB} -s ${prefix}${device.token} shell settings get global always_finish_activities`
```